savaslabs/savaslabs.github.io
# Issue #4889: Add page to style guide for blog posts

Fixes issue [#4889](https://pm.savaslabs.com/issues/4889)

## Summary of changes

1. Adds a page to the style guide for blog post guidelines and standards
1. Fills this page out and updates some of the information
1. Removes this information from the README
1. Removes this information from the blog post template and updates some info there
1. Improves styles for basic pages in the style guide (home and blog), improves active nav item behavior in the style guide, fixes some issues with the footer
1. Updates the pull request template to include items for blog posts

## Notes

Please suggest (or push) updates or additions to the style guide page! We can add to this on an ongoing basis. One thing I didn't include is a process for requesting a featured image.

## To test

- Pull this, `gulp clean && gulp serve`
- Visit http://local.savaslabs.com:3000/styleguide/blog.html
- Make sure the active nav items work in the style guide

### Pull request reviewer

As the reviewer, I have verified the following:
